exporter/azureblobexporter/README.md

```diff
@@ -36,18 +36,38 @@ The following settings can be optionally configured and have default values:
   - traces_format (default `2006/01/02/traces_15_04_05_{{.SerialNum}}.{{.FileExtension}}`): blob name format.
```

Comment:
Are these documented defaults still correct?

Reply:
good catch, I'll fix them in future PR.